assets/section-featured-blog.css

```diff
@@ -1,12 +1,14 @@
 .blog-placeholder {
-  margin: 0 1.5rem;
   background: rgb(var(--color-background));
+  width: 100%;
+  color: inherit;
+  position: relative;
 }
 
 @media screen and (min-width: 750px) {
   .blog-placeholder {
     text-align: center;
-    width: 50%;
+    width: 100%;
```

Comment:
Since you have already added `width: 100%` to `.blog-placeholder`(line 3) I don't think we need to add the same declaration for `@media screen and (min-width: 750px)` because in this case `width` will be inherited from line 3.

Reply:
Thank you! That is a good call out :)